org.aimodels.aiy.json

```diff
@@ -0,0 +1,48 @@
+{
+  "app-id": "org.aimodels.aiy",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "22.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "aiy",
+  "finish-args": [
+    "--share=network"
```

Comment:
so this displays nothing?

Reply:
Looking at https://docs.flatpak.org/en/latest/sandbox-permissions.html#standard-permissions

I didn't see anything else that would be used. 
Would I need to add:

```
--socket=x11 - show windows using X11
--socket=fallback-x11 
```
Mainly what it needs to to be able to write out to the tty / respond to command line requests. 
I did see that cli apps need to have the desktop entry, which I assume opens a terminal.  Would a permission be related to that?

Reply:
cli app are not the ideal use case for flatpak. They won't be displayed on flathub.org and they are more awkward to use.

Reply:
Hmm. when I wrote on /r/ubuntu that I had made a deb, it was down-voted and a comment telling me to make a flatpak was upvoted (by a few people), so trying to be responsive to user requests I've spent a couple days trying to figure this out. My assumption was that this was requested as would make it simple for users of other distros to interact with it. 

I could host it on my site and write a wrapper that gets copied to /usr/local/bin or something? Or is flatpak really not the right direction in general for a cli tool?